src/simulation/sensors/imuSensor/imuSensor.h

```diff
@@ -73,19 +81,19 @@ class ImuSensor: public SysModel {
     bool NominalReady;                  //!< -- Flag indicating that system is in run
     Eigen::Matrix3d PMatrixAccel;   //!< [-] Cholesky-decomposition or matrix square root of the covariance matrix to apply errors with
 	Eigen::Matrix3d AMatrixAccel;   //!< [-] AMatrix that we use for error propagation
-	Eigen::Vector3d walkBoundsAccel;//!< [-] "3-sigma" errors to permit for states
+	Eigen::Vector3d walkBoundsAccel;  //!< [-] Total error bounds for accelerometer states. @warning Will be renamed to errorBoundsAccel in December 2025
```

Comment:
Mm, this gives the user 1 year to know a change is coming, but not a true depreciation.  Can we create the new `errorBoundsAccel` variable now as a privat variable, create the suitable setter and getter, document, but leave the public `walkBoundsAccel`.  To see if the user sets it, could we default it to -1, and if anything else when we set `errorBoundsAccel` to this value?  This way old code doesn't break, but users have 1 year to upgrade to the new setter and getter?